appveyor.yml

```diff
@@ -0,0 +1,53 @@
+version: "{branch}-ci-{build}"
+os:
+- Visual Studio 2013
```

Comment:
curious. I'm afraid previously boost 1.59 caused me some problems - I'll look into this